pkg/tf/serialization/testdata/cluster.golden.tf

```diff
@@ -94,10 +94,6 @@ resource "google_container_cluster" "foo" {
   project         = "my-project"
   release_channel = {}
 
-  resource_labels = {
```

Comment:
why removing?

Reply:
So I believe we aren't removing them, they were never generated.  Our golden test was way too flexible..

I'll send a separate PR to split it out!